### PR TITLE
Add Glaze theme

### PR DESCRIPTION
**Pull request by CharlieS1103**

### CODE REVIEW

**Comment by morpheusthewhite** [comment]
Can you summarize the main changes from the default theme?

**Comment by CharlieS1103** [comment]
@morpheusthewhite Uh there's quite a bit but sure, for starters the side bar was more or less changed rather massively, with there being some separation between elements, the selected tab having a different shape and a gradient, as well as the removal of some clutter. There's also the changes to the playbar and playbar controls which make it look rather organized and again, reduce clutter. The episodes, liked song, and create playlist buttons had the labels removed and reformatted to be less distracting. Probably a bit more to come but that's what I can think of off the top of my head

File: THEMES.md [hunk 1]
```diff
@@ -74,6 +74,18 @@ Here you can find a preview of all the themes. Some of them may have different c
 
 ![light](Fluent/screenshots/light-1.png)
 
+## Glaze
+
+### Base
+<img src="https://github.com/CharlieS1103/Glaze-theme/blob/main/screenshots/base.png" alt="img" width="500px">
```
Comment (morpheusthewhite) [inline THEMES.md:80]: Do not use fixed-size images. Same in the README

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Comment by CharlieS1103** [comment]
@morpheusthewhite Done

File: Glaze/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Glaze
+
+## Screenshots
+
+### Base
+<img src="https://github.com/CharlieS1103/Glaze-theme/blob/main/screenshots/base.png" alt="img"> 
+
+
+### Dark
+<img src="https://github.com/CharlieS1103/Glaze-theme/blob/main/screenshots/dark.png" alt="img" ">
```
Comment (morpheusthewhite) [inline Glaze/README.md:10]: There is a syntax error here, the image is not displayed. I wonder why you didn't just use the Markdown directive for displaying the images

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Review by morpheusthewhite — APPROVED** [review]
